src/vision/calib/default.xml

```diff
@@ -0,0 +1,51 @@
+<Settings>
+    <!-- Number of inner corners per a item row and column. (square, circle) -->
+    <BoardSize_Width>8</BoardSize_Width>
+    <BoardSize_Height>5</BoardSize_Height>
+         
+    <!-- The size of a square in some user defined metric system (pixel, millimeter)-->
+    <Square_Size>97</Square_Size>
+                
+    <!-- The type of input used for camera calibration. One of: CHESSBOARD CIRCLES_GRID ASYMMETRIC_CIRCLES_GRID -->
+    <Calibrate_Pattern>"CHESSBOARD"</Calibrate_Pattern>
+                     
+     <!--  If true (non-zero) we flip the input images around the horizontal axis.-->
+     <Input_FlipAroundHorizontalAxis>0</Input_FlipAroundHorizontalAxis>
+                                                                
+     <!-- Time delay between frames in case of camera. -->
+     <Input_Delay>1</Input_Delay>  
+                                                                      
+     <!-- How many frames to use, for calibration. -->
+     <Calibrate_NrOfFrameToUse>100</Calibrate_NrOfFrameToUse>
+     <!-- Consider only fy as a free parameter, the ratio fx/fy stays the same as in the input cameraMatrix. 
+              Use or not setting. 0 - False Non-Zero - True-->
+     <Calibrate_FixAspectRatio> 1 </Calibrate_FixAspectRatio>
+     <!-- If true (non-zero) tangential distortion coefficients  are set to zeros and stay zero.-->
+     <Calibrate_AssumeZeroTangentialDistortion>0</Calibrate_AssumeZeroTangentialDistortion>
+     <!-- If true (non-zero) the principal point is not changed during the global optimization.-->
+     <Calibrate_FixPrincipalPointAtTheCenter> 1 </Calibrate_FixPrincipalPointAtTheCenter>
+                                                                                               
+     <!-- The name of the output log file. -->
+     <Write_outputFileName>"out_camera_data.yml"</Write_outputFileName>
+     <!-- If true (non-zero) we write to the output file the feature points.-->
+     <Write_DetectedFeaturePoints>1</Write_DetectedFeaturePoints>
+     <!-- If true (non-zero) we write to the output file the extrinsic camera parameters.-->
+     <Write_extrinsicParameters>1</Write_extrinsicParameters>
+     <!-- If true (non-zero) we show after calibration the undistorted images.-->
+     <Show_UndistortedImage>1</Show_UndistortedImage>
+     <!-- If true (non-zero) will be used fisheye camera model.-->
+     <Calibrate_UseFisheyeModel>1</Calibrate_UseFisheyeModel>
+     <!-- If true (non-zero) distortion coefficient k1 will be equals to zero.-->
+     <Fix_K1>0</Fix_K1>
+     <!-- If true (non-zero) distortion coefficient k2 will be equals to zero.-->
+     <Fix_K2>0</Fix_K2>
+     <!-- If true (non-zero) distortion coefficient k3 will be equals to zero.-->
+     <Fix_K3>0</Fix_K3>
+     <!-- If true (non-zero) distortion coefficient k4 will be equals to zero.-->
+     <Fix_K4>0</Fix_K4>
+     <!-- If true (non-zero) distortion coefficient k5 will be equals to zero.-->
+     <Fix_K5>1</Fix_K5>
```

Comment:
I don't know if the FileStorage object supports this, though it would make sense for it to. I'll look into it.